templates/edo/dumps.yaml

```diff
@@ -0,0 +1,387 @@
+          "permission": {{$permission_auth_manageKerberosMappings}},
+          "target": {{$wildcard}}
+        },
+        {
+          "principal": {{$serviceAccount_edgeDeploymentServiceAccount}},
+          "permission": {{$permission_configStore_readConfigForApp}},
+          "target": {{$application_gitRepoConfiguration}}
+        },
+        {
+          "principal": {{$serviceAccount_edgeDeploymentServiceAccount}},
+          "permission": {{$permission_configStore_readConfigForApp}},
+          "target": {{$application_edgeClusterConfiguration}}
+        },
+        {
+          "principal": {{$serviceAccount_edgeDeploymentServiceAccount}},
+          "permission": {{$permission_configStore_readConfigForApp}},
+          "target": {{$application_edgeClusterTemplate}}
+        },
+        {
+          "principal": {{$serviceAccount_edgeDeploymentServiceAccount}},
+          "permission": {{$permission_configStore_writeConfigForApp}},
+          "target": {{$application_generalObjectInformation}}
+        },
+        {
+          "principal": {{$serviceAccount_edgeDeploymentServiceAccount}},
+          "permission": {{$permission_configStore_writeConfigForApp}},
+          "target": {{$application_edgeClusterConfiguration}}
+        },
+        {
+          "principal": {{$serviceAccount_edgeDeploymentServiceAccount}},
+          "permission": {{$permission_configStore_manageObjects}},
+          "target": {{$serviceRequirement_edgeFluxAccount}}
+        },
+        {
+          "principal": {{$serviceAccount_edgeDeploymentServiceAccount}},
+          "permission": {{$permission_configStore_manageObjects}},
+          "target": {{$classDef_edgeCluster}}
+        },
+        {
+          "principal": {{$serviceAccount_edgeDeploymentServiceAccount}},
+          "permission": {{$permission_auth_manageACLsPerPermission}},
+          "target": {{$permission_git_pullFromRepo}}
+        },
+        {
+          "principal": {{$serviceAccount_edgeDeploymentServiceAccount}},
+          "permission": {{$permission_auth_manageGroups}},
+          "target": {{$serviceRequirement_edgeFluxRoles}}
+        },
+        {
+          "principal": {{$serviceAccount_edgeDeploymentServiceAccount}},
+          "permission": {{$permission_git_createRepo}},
+          "target": {{$serviceRequirement_edgeClusterRepositories}}
+        },
+        {
+          "principal": {{$serviceAccount_edgeDeploymentServiceAccount}},
+          "permission": {{$permission_git_pullFromRepo}},
+          "target": {{$serviceRequirement_edgeClusterRepositories}}
+        },
+        {
+          "principal": {{$serviceAccount_edgeDeploymentServiceAccount}},
+          "permission": {{$permission_git_pushToRepo}},
+          "target": {{$serviceRequirement_edgeClusterRepositories}}
+        }
+      ],
+      "groups": {
+          {{$permissionGroup_edo_all}}: [
+            {{$permission_edge_manageClusters}},
+            {{$permission_edge_manageSecrets}}
+          ]
```

Comment:
We also need, here, groups which fulfil the Service Requirements with the UUIDs for this deployment. So every Service Requirement UUID should be a group with at least one member filling in the UUIDs used locally. See the Auth service on my dev cluster.